File: server/query_db.mjs
import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient();

async function main() {
  // Check Product table
  const total = await prisma.product.count();
  const withCost = await prisma.product.count({ where: { purchasePrice: { not: null, gt: 0 } } });
  const withPrice = await prisma.product.count({ where: { salePrice: { not: null, gt: 0 } } });
  console.log("Product stats:", { total, withCost, withPrice, withoutCost: total - withCost });
  
  // Sample products
  const samples = await prisma.product.findMany({
    take: 10,
    select: { id: true, title: true, purchasePrice: true, salePrice: true, stock: true }
  });
  console.log("Sample products:", samples);
  
  // Orders
  const orderCount = await prisma.order.count();
  const orderSamples = await prisma.order.findMany({
    take: 5,
    select: { id: true, orderNo: true, total: true, status: true, marketplaceId: true }
  });
  console.log("Order count:", orderCount);
  console.log("Order samples:", orderSamples);
  
  // Marketplaces
  const mps = await prisma.marketplace.findMany({
    select: { id: true, name: true, key: true, active: true }
  });
  console.log("Marketplaces:", mps);
  
  await prisma.$disconnect();
}

main().catch(e => { console.error(e); process.exit(1); });

File: server/query_db.py
import sqlite3
conn = sqlite3.connect(r'C:\PROJE 1\DG-STOK-THEME-V1\server\dev.db')
cursor = conn.cursor()
cursor.execute("SELECT name FROM sqlite_master WHERE type='table'")
tables = cursor.fetchall()
print("Tables:", tables)

# Find Product table
product_table = None
for t in tables:
    if 'product' in t[0].lower():
        product_table = t[0]
        break

if product_table:
    cursor.execute(f"SELECT COUNT(*) as total, COUNT(purchasePrice) as with_cost, COUNT(salePrice) as with_price FROM {product_table}")
    print("Product stats:", cursor.fetchall())
    
    # Check sample of products
    cursor.execute(f"SELECT id, title, purchasePrice, salePrice, stock FROM {product_table} LIMIT 10")
    print("Sample products:", cursor.fetchall())
    
    # Check orders
    cursor.execute("SELECT name FROM sqlite_master WHERE type='table' AND name LIKE '%order%'")
    print("Order tables:", cursor.fetchall())

conn.close()